Sources/SwiftCompilerPluginMessageHandling/PluginMessages.swift

```diff
@@ -229,7 +229,8 @@ public enum PluginMessage {
     public var notes: [Note]
     public var fixIts: [FixIt]
 
-    internal init(
+    @_spi(PluginMessage)
+    public init(
```

Comment:
```suggestion
    public init(
```
All symbols in `@_spi` implicitly inherits it. Could you tell me what's using this?

Reply:
We're using this to emit diagnostics from `WasmInterceptingMessageHandler`:

https://github.com/apple/swift/blob/caa1a8ccd01eb568bc1b65007882bf8840e08ad6/tools/swift-plugin-server/Sources/swift-plugin-server/WasmMessageHandler.swift#L86-L100